skills/core_test.go

```diff
@@ -48,6 +48,8 @@ func (s *CoreSuite) TestValidateName(c *C) {
 		"",
 		// dashes alone are not a name
 		"-", "--",
+		// double dashes in a name are not allowed
+		"a--a",
```

Comment:
I wonder if its worth adding: "a-0a" and "a-b-c" to the list of valid names that are tested. But maybe not.